### PR TITLE
Update Dozer V2.1

### PR DESCRIPTION
**Pull request by Mortennn**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/dozer.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
 cask 'dozer' do
-  version '2.0.0'
-  sha256 '2aa0cd777cb37e7aaddd35d2e613b7bcfdfb1317fa13089de0914029d2750b85'
+  version '2.1.0'
+  sha256 'c4b496cdfa9336561a3ed5cfe4213708e93d9d3890b94c13e4d722e7f57c8675'
 
-  url "https://github.com/Mortennn/Dozer/releases/download/v#{version}/Dozer.#{version}.dmg"
-  appcast 'https://github.com/Mortennn/Dozer/releases.atom'
+  url "https://github.com/Mortennn/Dozer/releases/download/#{version}/Dozer.#{version}.dmg"
+  appcast 'https://raw.githubusercontent.com/Mortennn/Dozer/master/appcast.xml'
```
Comment (commitay) [inline Casks/dozer.rb]: Please use `https://github.com/Mortennn/Dozer/releases.atom`

File: Casks/dozer.rb [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
   name 'Dozer'
   homepage 'https://github.com/Mortennn/Dozer'
 
-  auto_updates true
-  depends_on macos: '>= :sierra'
+  auto_updates :yes
```
Comment (commitay) [inline Casks/dozer.rb]: `auto_updates :yes` isn't valid, please use `auto_updates true`

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/dozer.rb (hunk 1 above)
Comment (commitay) [reply Casks/dozer.rb]: We prefer to use the GitHub releases feed.

**Comment by Mortennn** [comment]
I still can't install Dozer with `brew cask install dozer`.
When will it be available?

**Pull request merged**